lua/dapui/render/canvas.lua

```diff
@@ -132,7 +132,8 @@ end
 ---Apply a render.canvas to a buffer
 ---@param state dapui.Canvas
 ---@param buffer number
-function M.render_buffer(state, buffer)
+---@param table expand_lines
+function M.render_buffer(state, buffer, expand_lines)
```

Comment:
All of the rendering decisions should be stored on the canvas object. Please add a method to set expand lines to enabled on the `Canvas` object, so then in this function we can do `if state.expand_lines.enabled`